<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapForThreePlayersClass</class>
 <widget class="QMainWindow" name="MapForThreePlayersClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MapForThreePlayers</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image:url(:/gui/mapBackground.png)</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: transparent
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Ro.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>951</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent
</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>670</y>
      <width>1131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>830</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>20</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>1043</x>
      <y>150</y>
      <width>20</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="zona1">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>110</y>
      <width>271</width>
      <height>188</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>242</width>
      <height>168</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona3">
    <property name="geometry">
     <rect>
      <x>1053</x>
      <y>80</y>
      <width>279</width>
      <height>218</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>298</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>297</y>
      <width>243</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona6">
    <property name="geometry">
     <rect>
      <x>1053</x>
      <y>298</y>
      <width>351</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>458</y>
      <width>500</width>
      <height>222</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona8">
    <property name="geometry">
     <rect>
      <x>802</x>
      <y>458</y>
      <width>251</width>
      <height>222</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent
</string>
    </property>
    <property name="text">
     <string>zona8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona9">
    <property name="geometry">
     <rect>
      <x>1053</x>
      <y>459</y>
      <width>557</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona10">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>680</y>
      <width>361</width>
      <height>158</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona11">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>681</y>
      <width>243</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona12">
    <property name="geometry">
     <rect>
      <x>1054</x>
      <y>680</y>
      <width>551</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona13">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>838</y>
      <width>210</width>
      <height>123</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona14">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>838</y>
      <width>243</width>
      <height>153</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona14</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zona15">
    <property name="geometry">
     <rect>
      <x>1053</x>
      <y>838</y>
      <width>451</width>
      <height>139</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string>zona15</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
